tests/test_loccalc.py

```diff
@@ -0,0 +1,24 @@
+#  Copyright (c) 2021 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from uberpoet.loccalc import LOCCalculator
+
+
+class TestLOCCalculator(unittest.TestCase):
+
+    def test_calculate_swift_loc(self):
+        loc_calc = LOCCalculator()
+        self.assertEqual(loc_calc.calculate_swift_loc(), 132)
```

Comment:
Err, how does it count / calculate something when you don't specify a file?  The test is confusing when you first read it.   We should make it take in a file to make it a less surprising piece of code.

Reply:
yeah I see your point. The original intention was that the method itself was conveying a "template Swift loc".

I have changed this now to accept a filegenerator to calculate loc for instead.